samples/NugetDeployment/HttpListenerExample.csproj

```diff
@@ -0,0 +1,18 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net5.0</TargetFramework>
```

Comment:
Is this correct? We never specify both `TargetFramework` and `TargetFrameworks` in other projects. Will this act as the default if not specified (like `dotnet run -f net5.0`) while allowing to override (`dotnet run -f net48`)?